client/src/components/FlightAccordion.js

```diff
@@ -0,0 +1,247 @@
+/** @format */
+
+import React, { useState } from "react";
+import {
+	Accordion,
+	AccordionSummary,
+	AccordionDetails,
+	Grid,
+	Typography,
+	Divider,
+	Button,
+} from "@material-ui/core";
+import Timeline from "@material-ui/lab/Timeline";
+import TimelineItem from "@material-ui/lab/TimelineItem";
+import TimelineSeparator from "@material-ui/lab/TimelineSeparator";
+import TimelineConnector from "@material-ui/lab/TimelineConnector";
+import TimelineContent from "@material-ui/lab/TimelineContent";
+import TimelineDot from "@material-ui/lab/TimelineDot";
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
+import DepartLogo from "../assets/images/depart.png";
+import TurkLogo from "../assets/images/turk-logo.png";
+import useStyles from "../styles/FlightAccordion";
+
+const FlightAccordion = ({ quote, carrier, date }) => {
+	const classes = useStyles();
+	const [expand, setExpand] = useState(false);
+
+	const showDetails = (event, expanded) => {
+		if (expanded) {
+			setExpand(true);
+		} else {
+			setExpand(false);
+		}
+	};
+
+	return (
+		<>
+			<Accordion
+				onChange={showDetails}
```

Comment:
Does this work? You're not passing in the `expanded` parameter you defined in the `showDetails` function. Check out the [`Accordion` docs](), see how it's implemented in the examples, particularly with the `expanded` attribute.

Reply:
Yes, it works. The 'expanded' parameter is used in the if statement.